src/select/select.js

```diff
@@ -334,11 +336,17 @@ class Select extends React.Component<PropsT, StatelessStateT> {
 
   getDropDown() {
     const {overrides, type, rows, selectedOptions} = this.props;
+    let maxHeight = this.props.maxHeight;
+    if (__BROWSER__ && parseInt(maxHeight) > window.innerHeight) {
```

Comment:
Should we have some check to make sure we're dealing here with `maxHeight` pixels value?

Reply:
added